### PR TITLE
Migrating to Siddhi version 4.0.0-M7

### PR DESCRIPTION
**Pull request by dilini-muthumala**

### CODE REVIEW

File: component/findbugs-exclude.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tishan89) [inline component/findbugs-exclude.xml]: do we need the pre-emble

File: component/pom.xml [hunk 2]
```diff
@@ -72,8 +73,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.0</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```
Comment (tishan89) [inline component/pom.xml]: add meta-inf resource inclusion to bundle plugin

Comment (dilini-muthumala) [inline component/pom.xml]: Added in 	429a3eb

File: component/findbugs-exclude.xml (hunk 1 above)
Comment (dilini-muthumala) [inline component/findbugs-exclude.xml]: This is an XML document, so no harm to have it right? :-)

Comment (tishan89) [inline component/findbugs-exclude.xml]: ok

**Pull request merged**